.gitignore

```diff
@@ -31,6 +31,7 @@ Referenced Packages
 **/.sfdx/
 **/.vscode/
 **/.idea/
+**/.mypy_cache/
```

Comment:
@KietTruongSFDO I think this is also something that is unique to your dev environment. If you need it to be in _your_ gitignore, you can add it to your global gitignore file.

Reply:
Same as above, I didn't make this change and I think the branches are out of sync. I'll pull and update again.